lib/stdlib/src/beam_lib.erl

```diff
@@ -832,8 +832,7 @@ symbol(_, AT, I1, I2, _I3, _Cnt) ->
     {atm(AT, I1), I2}.
 
 atm(AT, N) ->
-    [{_N, S}] = ets:lookup(AT, N),
-    S.
+    ets:lookup_element(AT, N, 2).
```

Comment:
Not fully equivalent.

Reply:
This ETS table is short-lived and only writeable by the owner. I find it very unlikely that the tuple size assertion would ever find a bug.